src/legal-officer/transaction-protection/ContextualizedLocDetails.tsx

```diff
@@ -97,7 +97,11 @@ export default function ContextualizedLocDetails() {
     if(loc.locType === 'Transaction') {
         locTabTitle = "Logion Officer Case - Transaction";
     } else {
-        locTabTitle = "Logion Officer Case - Identity";
+        if (isLogionIdentityLoc(loc)) {
+            locTabTitle = "Logion Officer Case - Logion Identity";
+        } else {
+            locTabTitle = "Logion Officer Case - Polkadot Identity";
```

Comment:
We just had a discussion with Elie: in the case of a regular LOC, he prefers that we do not mention Polkadot. The only exception is on the identity LOCs screen in the title above the table.

Reply:
Fixed. Commit amended et force pushed: https://github.com/logion-network/logion-wallet/compare/b982ca21281d36c41831a922e2c2c6c2e45fb4dc..0117c0f03462711b93be6c1b0a1a90bf7eeacc9b